docs/FeatureManagement/Samples/RequirementType.sample.json

```diff
@@ -0,0 +1,144 @@
+{
+    "feature_management": {
+        "feature_flags": [
+            {
+                "id": "FirstOrFeatureFlag",
+                "description": "A feature flag that has multiple filters, but doesn't specify any requirement type, which is the default. Will always return true.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Tue, 27 Jun 2023 06:00:00 GMT"
+                            }
+                        },
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Thu, 29 Jun 2023 07:00:00 GMT",
+                                "End": "Wed, 30 Aug 2023 07:00:00 GMT"
+                            }
+                        }
+                    ]
+                }
+            },
+            {
+                "id": "SecondOrFeatureFlag",
+                "description": "Same as Iota, but filter order is switched. Will always return true.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Thu, 29 Jun 2023 07:00:00 GMT",
+                                "End": "Wed, 30 Aug 2023 07:00:00 GMT"
+                            }
+                        },
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Tue, 27 Jun 2023 06:00:00 GMT"
+                            }
+                        }
+                    ]
+                }
+            },
+            {
+                "id": "FristRequirementTypeAnyFeatureFlag",
```

Comment:
```suggestion
                "id": "AnyRequirementTypeFeatureFlag1",
```

Reply:
The reason to use AnyRequirementType instead of RequirementTypeAny is because I want to align with the previous "DefaultRequirementTypeFeatureFlag".

But I am ok with RequirementTypeAny

Reply:
I don't like `AnyRequirementType` as it reads as "any requirement type` not, "the value of the requirement type is any".

Reply:
Make sense.